internal/functions/deploy/deploy.go

```diff
@@ -18,7 +18,10 @@ import (
 	"github.com/supabase/cli/pkg/api"
 )
 
-const EszipContentType = "application/vnd.denoland.eszip"
+const (
+	eszipContentType = "application/vnd.denoland.eszip"
+	supabaseDirPath  = "supabase"
```

Comment:
Let's move this to misc https://github.com/supabase/cli/blob/main/internal/utils/misc.go#L86

Reply:
Left eszipContentType here since it's not used elsewhere.